.github/workflows/npm-publish.yml

```diff
@@ -29,6 +29,11 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event_name == 'release' && github.event.action == 'created'
     steps:
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
is there a difference between `github.token` and `secrets.GITHUB_TOKEN`?

Reply:
Good catch! I don't believe so. will update for consistency